packages/patterns/src/keys/checkKey.js

```diff
@@ -126,14 +128,15 @@ export const checkKey = (val, check) => {
 harden(checkKey);
 
 /**
- * @param {Passable} val
+ * @param {Key} val
```

Comment:
Shouldn't this be

```suggestion
/**
 * @param {any} val
 * @returns {val is Key}
 */
```
?

Reply:
indeed